RobotCode/src/main/java/frc/lib/modules/shootermount/ShooterMountSubsystem.java

```diff
@@ -107,6 +108,7 @@ public ShooterMountSubsystem()
 				() -> (gyro.getRoll().getValueAsDouble() / 360));
 	}
 
+	//If a motor gets reset this tells to continue sending only what it needs to send the the CAN bus
```

Comment:
Move this comment inside of periodic, periodic can be used for more than just this.